### ISSUE TITLE
Can you show a working demo

### ISSUE THREAD

**Issue by NeverInAsh** [Sashorg/voicebot_unity#1]
Hi Sash,
this is something very new and seems very interesting.

Do you have something like a tutorial or walkthrough to understand setup and running this project.

**Comment by Sashorg** [comment]
Hi, actually what I have found out that generation time for both response , when using multiple characters is too slow. It is because RASA is expected to have one character behaviour. So I actually stopped the project because of this reason.

Среда, 10 февраля 2021, 14:54 +03:00 от NeverInAsh <notifications@github.com>:
>Hi Sash,
>this is something very new and seems very interesting.
>Do you have something like a tutorial or walkthrough to understand setup and running this project.
>—
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly,  view it on GitHub , or  unsubscribe .

-- 
саша сухов

**Comment by NeverInAsh** [comment]
Would it be possible for you to make it work for a single character behavior ?

**Comment by Sashorg** [comment]
Yes, I tried it and response is much faster as it doesn't access another python script. However my first idea was considered many characters, so there was no sense of developing something with one character. 


Среда, 10 февраля 2021, 15:11 +03:00 от NeverInAsh <notifications@github.com>:
>Would it be possible for you to make it work for a single character behavior ?
>—
>You are receiving this because you commented.
>Reply to this email directly,  view it on GitHub , or  unsubscribe .

-- 
саша сухов
